doc/book/zend.ldap.introduction.md

```diff
@@ -0,0 +1,195 @@
+# Introduction to Zend\\Ldap
+
+`Zend\Ldap\Ldap` is a class for performing *LDAP* operations including but not limited to binding,
+searching and modifying entries in an *LDAP* directory.
+
+## Theory of operation
+
+This component currently consists of the main `Zend\Ldap\Ldap` class, that conceptually represents a
+binding to a single *LDAP* server and allows for executing operations against a *LDAP* server such
```

Comment:
Note to @heiglandreas: Check how a space-separated list of multiple LDAP-Servers is handled